py-bindings/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "cmake>=3.18",
+    "ninja",
+    "pygccxml",
+    "numpy",
+    # Need latest commit for this PR with Mac fixes:
+    # https://github.com/ompl/pyplusplus/pull/1
+    "pyplusplus @ git+https://github.com/ompl/pyplusplus",
```

Comment:
This URL can go away if a new pyplusplus release is made to PyPI.

Reply:
A new pyplusplus tag has been made, 1.8.6

Reply:
PyPI is still showing 1.8.5. I think the release needs to be made there before updating this line.